<?xml version="1.0"?>
<launch>
  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
    <arg name="align_depth" value="true"/>
  </include>
  <node name="best_prediction" pkg="raiv_research" type="node_best_prediction.py">
    <param name="invalidation_radius" value="150"/>  <!-- radius to invalidate the predictions (in pixel) -->
    <param name="image_topic" value="/usb_cam/image_raw"/>
    <param name="crop_width" value="56"/>  <!-- size of cropped image -->
    <param name="crop_height" value="56"/>
    <param name="model_name" value="METTRE ICI LE CHEMIN DU FICHIER CKPT"/>
  </node>
  <node name="visu_prediction" pkg="raiv_research" type="node_visu_prediction.py" output="screen">
    <param name="prediction_threshold" value="0.5"/>
  </node>
</launch>
